internal/acceptance_test/resource_host_test.go

```diff
@@ -20,28 +21,48 @@ const (
 	hostStatePollCount = 4
 )
 
-func TestAccResourceHost_Basic(t *testing.T) {
+func TestAccResourceHost_Async(t *testing.T) {
 	resource.Test(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
 		Providers:    testAccProviders,
 		CheckDestroy: resource.TestCheckFunc(func(s *terraform.State) error { return testAccCheckHostDestroy(t, s) }),
 		Steps: []resource.TestStep{
 			// host create step
 			{
-				Config: testAccCheckHostBasic(),
+				Config: testAccCheckHostBasic(true),
```

Comment:
Define isAsync = true and isNotAsync = false in the const block above. Will make these easier to read. It'd be good to have in comment somewhere that async is the default/old behavior. Not sure what the best place is to document that.

Reply:
resource_host.go defines hHostActionAsync with a description